dev/TeachingTip/TeachingTip.cpp

```diff
@@ -1104,7 +1104,11 @@ void TeachingTip::StartContractToClose()
         if (m_beakOcclusionGrid)
         {
             m_beakOcclusionGrid.get().StartAnimation(m_contractAnimation.get());
-            m_beakOcclusionGrid.get().StartAnimation(m_contractElevationAnimation.get());
+            m_isContractAnimationPlaying = true;
+        }
+        if (m_contentRootGrid)
+        {
+            m_contentRootGrid.get().StartAnimation(m_contractElevationAnimation.get());
             m_isContractAnimationPlaying = true;
```

Comment:
Setting this variable to true in 3 consecutive branches feels weird. Seems like it should just be set after starting all the animations.

Reply:
These are all template parts, if we don't have any of them we wont have started any animations so the flag would need to be false.